packages/amplify-graphql-schema-test-library/src/index.ts

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable no-bitwise */
+export const enum TransformerVersion {
```

Comment:
Discussed offline, but I'd recommend dropping a comment on these enums that we're representing them as bitmaps since you can select multiple target versions or platforms in a request.